src/app/adr-details-view/adr-details-view.component.ts

```diff
@@ -0,0 +1,53 @@
+import {Component, OnInit, ViewEncapsulation, ChangeDetectionStrategy, Input} from '@angular/core';
+import {TechRecordHelpersService} from '@app/technical-record/tech-record-helpers.service';
+import {DownloadDocumentFileAction} from '@app/adr-details-form/store/adrDetails.actions';
+import {select, Store} from '@ngrx/store';
+import {IAppState} from '@app/adr-details-form/store/adrDetailsForm.state';
+import {Observable} from 'rxjs';
+
+@Component({
+  selector: 'vtm-adr-details-view',
+  templateUrl: './adr-details-view.component.html',
+  styleUrls: ['./adr-details-view.component.scss'],
+  encapsulation: ViewEncapsulation.Emulated,
```

Comment:
i don't think we need those. emulated is the default option
https://angular.io/api/core/ViewEncapsulation

Reply:
This is the way Angular generated my component

Reply:
you can choose a path when you create angular component
https://stackoverflow.com/questions/42144953/how-can-i-change-the-path-of-component-generation-to-current-path-in-angular-cli